lib/extensions.js

```diff
@@ -89,8 +89,9 @@ function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
          package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
```

Comment:
Sure we can; but this changes the meaning of `SASS_BINARY_URL` variable.

If somebody set it like this

````
SASS_BINARY_URL=http://somehost:8080/local/freebsd-x64-14_binding.node
````

it will no longer work, instead they would need to re-organize their files under `'v'` + `package.version` to make this work again.

We can deprecate the `SASS_BINARY_URL` variable if we want, but let's give a new one a new name and let's warn users that the old one will no longer work.

This can be very confusing when somebody is upgrading a larger set of machines with `npm`...